src/quemb/molbe/fragment.py

```diff
@@ -137,6 +138,7 @@ def __init__(
             self.center_atom = fragment_map.center_atoms
             self.centerf_idx = fragment_map.centerf_idx
             self.ebe_weight = fragment_map.ebe_weights
+            self.dnames = fragment_map.dnames
```

Comment:
Is `dnames` just the order within the list of fragments (i.e. fragment 0, fragment 1, etc)? It is a bit unclear how where this is being used, since `dnames` seems to only exist in `graphgen`. (i.e. if `autogen` is called, this would be missing, right?)